cmd/generator/main.go

```diff
@@ -259,18 +265,52 @@ func getReleasePlugins(ctx context.Context, client *github.Client, repositoryNam
 		plugin.Signatures = signatures
 
 		if plugin.Manifest == nil {
-			return nil, fmt.Errorf("failed to find plugin manifest for release %s", releaseName)
+			logger.Warnf("failed to find plugin manifest for release %s", releaseName)
+			continue
 		}
 
-		// Ignore if we have the latest plugin version for this server version
-		if serverVersions[plugin.Manifest.MinServerVersion] {
-			continue
+		if minServerVersionsSeen[plugin.Manifest.MinServerVersion] != nil {
+			if plugin.Manifest.Version == "" {
+				logger.Warnf("version is empty for manifest.Id %s", plugin.Manifest.Id)
+				continue
+			}
+
+			p := minServerVersionsSeen[plugin.Manifest.MinServerVersion]
```

Comment:
Distinguishing `p` from `plugin` is tricky -- can we call this `lastSeenPlugin` or some such instead?